user-service/model/jwt.js

```diff
@@ -4,13 +4,17 @@ import redis from 'redis';
 
 const EXPIRATION = 86400; // 24hrs
 
-const client = redis.createClient({
-    socket: {
+const redisClientOptions = process.env.ENV
```

Comment:
Is the comparison of ENV to LOCAL or PROD missing here?

Reply:
Good catch thanks!